### DISCUSSION TITLE
Plan ID field on In Network Rate Field

### DISCUSSION THREAD

**Discussion by Corrigall** [CMSgov/price-transparency-guide#286] [Q&A] [answered]
@shaselton-usds 

Hello,

I am hoping to confirm my understanding. The following definition is provided for the field PLAN_ID: 
The 14-digit Health Insurance Oversight System (HIOS) identifier, or, if the 14-digit HIOS identifier is not available, the 5-digit HIOS identifier, or if no HIOS identifier is available, the Employer Identification Number (EIN)for each plan or coverage offered by a plan or issuer.

Based off previous webinars and other vendor meetings I have attended, I am under the impression that the PLAN_ID no longer has to be classified with the HIOS, but we can classify it along the provider group or member group level (whatever reflects common rates better).

Can someone please confirm this for me? Using the HIOS brings the files to the group grain level almost and would require thousands of files, all very repetitive. We believe we can group rates more accurately and reduce the amount of files we need to provide.

If we can group further up (i.e. at the member group level or provider group level or another attribute that bundles groups better than HIOS ), which file (if any) are we referencing the list of groups under a PLAN_ID?

Thank you for any feedback.

**Comment by shaselton-usds** [comment, marked as answer]
@Corrigall Wrapping up some old threads -- thanks for the patience.

The HIOS requirements were updated from 14-digit to 10-digits - you can see that here: https://github.com/CMSgov/price-transparency-guide/pull/447

The `plan_id` still requires the HIOS if available. Here is the documentation: "The 10-digit Health Insurance Oversight System (HIOS) identifier, or, if the 10-digit HIOS identifier is not available, the 5-digit HIOS identifier, or if no HIOS identifier is available, the Employer Identification Number (EIN)for    each plan or coverage offered by a plan or issuer."

You can also leverage the [table of contents](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/table-of-contents) file to cut down on all of the duplicate negotiated rates across plans/products. 

Hopefully that helps.
